source/less/components/programs.less

```
@@ -12,14 +25,117 @@
 
 .programs__item {
   margin-bottom: 20px;
+  padding: 24px 20px 0;
+
+  display: grid;
+  grid-template-columns: 50px 1fr;
+  column-gap: 20px;
+
   background-color: @bg-color-lighter;
 
   @media (min-width: @tablet-width) {
     margin-bottom: 30px;
+    padding: 77px 60px 34px 52px;
+
+    grid-template-columns: 1fr 200px;
+    column-gap: 54px;
+    grid-template-areas: "title icon"
+      "des icon"
+      "link icon";
   }
 
   @media (min-width: @desktop-width) {
+    padding: 39px 50px 37px 52px;
     width: 570px;
+
+    grid-template-columns: 100px 1fr;
+    column-gap: 62px;
+    grid-template-areas: "icon title"
+      "des des"
+      "link link";
+
+    box-sizing: border-box;
+  }
+}
+
+.programs__mark {
+  position: relative;
+
+  width: 50px;
+  height: 50px;
+
+  background-color: @element-basic-color;
+  border-radius: 50%;
+
+  &::before {
+    content: "";
+    position: absolute;
+  }
+
+  &--slim::before {
+    top: -2px;
+    left: 10px;
+    width: 36.02px;
+    height: 50px;
+    background: url("../../img/cat-slim.svg") no-repeat;
```

во всех файлах стилей так должно быть
```suggestion
    background: url("../img/cat-slim.svg") no-repeat;
```